var/spack/repos/builtin/packages/nwchem/package.py

```diff
@@ -81,9 +86,9 @@ def install(self, spec, prefix):
         use_32_bit_lin_alg = True
 
         if use_32_bit_lin_alg:
-            args.extend(["USE_64TO32=y", "BLAS_SIZE=4", "LAPACK_SIZE=4", "SCALAPACK_SIZE=4"])
+            args.extend(["USE_64TO32=y", "BLAS_SIZE=4", "SCALAPACK_SIZE=4"])
         else:
-            args.extend(["BLAS_SIZE=8", "LAPACK_SIZE=8" "SCALAPACK_SIZE=8"])
+            args.extend(["BLAS_SIZE=8", "SCALAPACK_SIZE=8"])
```

Comment:
Can you explain why you removed the options? Does it hold also for previous versions?

Reply:
`LAPACK_SIZE` has never been used. It applies to all NWChem versions